pkgs/by-name/me/megapixels/search-for-postprocessors-in-NixOS-specific-global-location.patch

```diff
@@ -0,0 +1,60 @@
+--- a/src/process_pipeline.c
++++ b/src/process_pipeline.c
+@@ -171,7 +171,15 @@ mp_process_find_all_processors(GtkListStore *store)
+                         store, &iter, 0, buffer, 1, "(system) postprocess.sh", -1);
+         }
+ 
+-        // Check packaged /usr/share/megapixels/postprocess.sh
++        // Check system /run/current-system/sw/share/megapixels/postprocess.sh
++        sprintf(buffer, "/run/current-system/sw/share/megapixels/postprocess.sh");
```

Comment:
I don't think that's a good idea. It encourages installing packages with conflicting paths, e.g. megapixels and another postprocessing script. Which one will be available at `/run/current-system/sw/share/megapixels/postprocess.sh`?

Reply:
Yeah, that's the downside. The upside is that if the user selects this postprocessor in the settings, the setting will keep working even if the Nix store hash of the script changes. I'm in favor of keeping it this way. What do you think?

Reply:
I think we should take `postprocessor.sh` from megapixels and only allow other packages to put files in `postprocessor.d/`

Reply:
@donovanglover what do you think?

Reply:
`postprocessor.d/` seems reasonable. I like the home-manager solution the most since it's declarative

Reply:
Done.